packages/core/src/tracing/sdk/sdk.js

```diff
@@ -8,6 +8,7 @@
 const deepMerge = require('../../util/deepMerge');
 const tracingUtil = require('../tracingUtil');
 const constants = require('../constants');
+const cls = require('../cls');
```

Comment:
_issue(blocking):_ We must not require `cls` unconditionally, see above.

Reply:
Will use https://github.com/instana/nodejs/pull/565#discussion_r912768179